game_event/test_game_event_views.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from django.urls import reverse
+from .models import GameEvent, Court
+from decimal import Decimal
+from .views import game_events
+from django.utils import timezone
+
+
+@pytest.mark.django_db
+class TestGameEventServerResponses:
+    @pytest.fixture
+    def saved_court(self):
+        court = Court.objects.create(x=Decimal('11'), y=Decimal('22'),
+                                     city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
+        return court
+
+    @pytest.fixture
+    def saved_game_event(self, saved_court):
+        game_event = GameEvent.objects.create(
+            id=1000,
+            time=timezone.now() - timezone.timedelta(days=1),
+            level_of_game=5,
+            min_number_of_players=4,
+            max_number_of_players=10,
+            court=saved_court,
+            ball_game='Basketball'
+        )
+        return game_event
+
+    def test_success_loading_site(self, client):
+        response = client.get('/game-events/')
+        assert response.status_code == 200
+
+    def test_fail_loading_site(self, client):
+        response = client.get('/game-eventsss/')
+        assert response.status_code == 404
+
+    def test_valid_min_player_filter(self, client, saved_game_event):
```

Comment:
I think this test and the next ones need to be renamed. The way I see it, those tests are not about valid and invalid inputs. They are about filters that will or will not include your event. The tests are an important tool for clarifying the code, so if you call something invalid, it makes me think what makes this value invalid. In this case, I think those values are valid - they just don't return our event.

Reply:
Thanks for the reference. I was thinking about it while writing the code, something felt wrong for me, but I couldn't put my finger on it. Now I got it. Thanks.